test/integration/imagechange_buildtrigger_test.go

```diff
@@ -61,6 +62,25 @@ func TestSimpleImageChangeBuildTriggerFromImageStreamTagDockerWithConfigChange(t
 func TestSimpleImageChangeBuildTriggerFromImageStreamTagCustom(t *testing.T) {
 	defer testutil.DumpEtcdOnFailure(t)
 	projectAdminClient := setup(t)
+
+	clusterAdminClient, err := testutil.GetClusterAdminClient(testutil.GetBaseDir() + "/openshift.local.config/master/admin.kubeconfig")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	clusterRoleBindingAccessor := policy.NewClusterRoleBindingAccessor(clusterAdminClient)
+	subjects := []kapi.ObjectReference{
+		{
+			Kind: authorizationapi.SystemGroupKind,
+			Name: bootstrappolicy.AuthenticatedGroup,
+		},
+	}
+	options := policy.RoleModificationOptions{
+		RoleNamespace:       testutil.Namespace(),
+		RoleName:            bootstrappolicy.BuildStrategyCustomRoleName,
+		RoleBindingAccessor: clusterRoleBindingAccessor,
+		Subjects:            subjects,
+	}
+	options.AddRole()
```

Comment:
Need to wait for role change to become effective (see other integration tests for example)